qiskit/algorithms/optimizers/de.py

```diff
@@ -0,0 +1,134 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Differential Evolution (DE) optimizer"""
+
+from typing import Callable, Tuple, List, Dict, Union
+
+import numpy as np
+from scipy.optimize import differential_evolution
+from .optimizer import Optimizer, OptimizerSupportLevel
+
+
+class DE(Optimizer):

```

Comment:
Is `DE` a standardized name that everyone uses? Or could we rename this to `DifferentialEvolution`? IMO that would be clearer and more in-line with how we name other objects in the algorithms module.

Reply:
@Cryoris I think we change it to `DifferentialEvolution` for clarity. Thanks for the suggestion!

Reply:
Fixed!

Reply:
No, I'd just stick with `DifferentialEvolution`. These are only uppercase because they are all abbreviations. I think we should add a `GradientDescent` in future too, which also won't be uppercase.

Reply:
Fixed!